DAQIANN-RESUME

DAQIANN opened the pull request:
Couldn't quite get the formatting in correctly, but this is mostly what I have on my resume. I don't have the github and linkedin links in my actual resume, Should I add this in?

DAQIANN committed:
Update on resume.md

Couldn't quite get the formatting in correctly, but this is mostly what I have on my resume. I don't have the github and linkedin links in my actual resume, Should I add this in?

DAQIANN committed:
Merge pull request #1 from DAQIANN/DAQIANN-resume

Update on resume.md

alextakahashi commented on resume.md:
```diff
@@ -1,49 +1,47 @@
+**University of Illinois Urbana Champaign** *May 2023* *GPA 3.89* *Dean's List*
+* Bachelor of Science in Mathematics and Computer Science 
+* Minor in Statistics 
+* **Coursework**: Data Structures, Numerical Methods I, Software Design Studio, Discrete Structures, Introduction to Computer Science, Linear Algebra, Statistical Analysis, Statistics and Probability I
```
Would use the generic name for these classes / would remove introduction classes
* Numerical Methods I (not sure what this is)
* Software Design Studio (not sure what this is)
* ~~Introduction to Computer Science~~

alextakahashi commented on resume.md:
```diff
@@ -1,49 +1,47 @@
-* **Cal Hacks 7.0 (remote)** - Berkeley, CA ... *Month 20XX*
-* **Cal Hacks 6.0** - Berkeley, CA ... *Month 20XX*
-* **WWDC 2019** - San Jose, CA ... *Month 20XX*
+**CS 128 Course Assistant**: *September 2020 - Present*
```
Use the generic name for this - outside of your school no one knows what this is.

Would update the following
* Company --> University of Blah
* City, State --> University's City / State

^ this will also help utilizing more white space in your resume

alextakahashi replied:
Is there a link to online course material?

alextakahashi commented on resume.md:
```diff
@@ -1,49 +1,47 @@
-* **Cal Hacks 6.0** - Berkeley, CA ... *Month 20XX*
-* **WWDC 2019** - San Jose, CA ... *Month 20XX*
+**CS 128 Course Assistant**: *September 2020 - Present*
+* Course assistant for CS 128, a new core Computer Science class to be implemented Spring 2021
```
Be specific + use action verbs.  Are you a founding member?  What is your role as course assistant?  How does this highlight your technical abilities?  How many people are going to be in the course? Is this upper division?

alextakahashi commented on resume.md:
```diff
@@ -1,49 +1,47 @@
+**CS 128 Course Assistant**: *September 2020 - Present*
+* Course assistant for CS 128, a new core Computer Science class to be implemented Spring 2021
+* Developed engaging assignments in C++ for future students
+* Worked cohesively with other assistants to produce the course material
```
This section should be pivoted to be more technical.  A bit hand wavy

alextakahashi commented on resume.md:
```diff
@@ -1,49 +1,47 @@
+* Worked cohesively with other assistants to produce the course material
+
+**Sandia National Laboratories Space Blimp Challenge**: *November 2019*
+* Led team that developed new software that improves the management of experimental flight data 
```
+1 add who hosted the event (I'm assuming its some association of some sort) + location.

Be specific - what kind of software what kind of tools, **why** does this matter?  How does this effect business metrics by using your software?

alextakahashi commented on resume.md:
```diff
@@ -1,49 +1,47 @@
+
+**Sandia National Laboratories Space Blimp Challenge**: *November 2019*
+* Led team that developed new software that improves the management of experimental flight data 
+* Effectively utilized Java to parse through given data for maximum time efficiency
```
How much data?  What methods / algorithms did you use to get to where you needed to be?   How well did you perform compared to other folks in the challenge? 

Is this code on github?

alextakahashi commented on resume.md:
```diff
@@ -1,49 +1,47 @@
+* Effectively utilized Java to parse through given data for maximum time efficiency
+
+**Alameda County and Engineering Fair**: *April 2018*
+* Presented a research study focused on Artificial Intelligence technologies and Machine Learning to industry professionals
```
Be specific, what was the research study?  I need to better understand your personal role in this.  What did you do as an independent contributor to this project?

alextakahashi commented on resume.md:
```diff
@@ -1,49 +1,47 @@
+
+**Alameda County and Engineering Fair**: *April 2018*
+* Presented a research study focused on Artificial Intelligence technologies and Machine Learning to industry professionals
+* Analyzed Artificial Intelligence services such as Amazon Rekognition Services, Google Cloud Vision Service, Microsoft’s Cognitive Services Vision and Face on their image correctness, accuracy, and comprehensiveness 
```
hand wavy, be specific to what end were you using these services?

alextakahashi commented on resume.md:
```diff
@@ -1,49 +1,47 @@
+**Special Interest Group Programing Languages (SIGPLAN)**: *October 2019 - Present*
+* Worked with students to reinforce knowledge of several high-level programming languages and algorithms 
+
+## SKILLS
```
My two cents - add skills to the top this will help the reader understand what tools you are aware about.  Experience should emphasize *how* you use these tools

alextakahashi replied:
would add any version control experience you have under your belt, git

^ consider putting this into a bucketed approach

* **Programming Languages**: Java, C, ...
* **Languages**: Mandarin Chinese (Fluent)
* **Enterprise Software (?)** Office, Drive, blah


```

DAQIANN replied:
Can I pivot the Progamming Languages section to just Programming? And include git into this as I don't think I have enough to fill a separate Version Control section

alextakahashi commented on resume.md:
```diff
@@ -1,49 +1,47 @@
+
+## LEADERSHIP/ACTIVITIES
+**Illinois Formula Electric**: *August 2020 - Present*
+* Member of the Data Acquisition Analysis (DAQA) subsystem
```
How did you contribute and why does this matter?

alextakahashi commented on resume.md:
```diff
@@ -1,49 +1,47 @@
+## LEADERSHIP/ACTIVITIES
+**Illinois Formula Electric**: *August 2020 - Present*
+* Member of the Data Acquisition Analysis (DAQA) subsystem
+* Worked collaboratively with clubmates to enhance data visualization by using Python
```
Be more specific, did you use a specific library/module to accomplish this?  How much did you improve?  Did you create graphs / diagrams?  Anything online that we can see?

alextakahashi commented on resume.md:
```diff
@@ -1,49 +1,47 @@
+
+**Special Interest Group in Security (SIGPWNY)**: *August 2020 - Present*
+* Engaged in presentations focused around information security fundamentals
+* Competed in Capture The Flag cyber competitions
```
I don't know what this is ^ explain the technical challenges behind this

alextakahashi commented on resume.md:
```diff
@@ -1,49 +1,47 @@
+* Worked collaboratively with clubmates to enhance data visualization by using Python
+
+**Special Interest Group in Security (SIGPWNY)**: *August 2020 - Present*
+* Engaged in presentations focused around information security fundamentals
```
Vague - what does it mean to engage in presentations?

alextakahashi commented on resume.md:
```diff
@@ -1,49 +1,47 @@
+**Special Interest Group Artificial Intelligence and Data Analytics (SIGAIDA)**: *October 2019 - Present*
+* Participated in comprehensive workshops focused around the applications of Machine Learning and Artificial Intelligence within the tech world
+* Gained experience with the use of PyTorch for neural network training 
+* Managing NLP Chatbot project written in Python
```
be more specific - talk more about the tools you used to make this chatbot.

alextakahashi reviewed (changes requested):
Daniel - this is a good start.  I've left comments + suggestions below.

In general
* **Highlight your Technical Background**: Dive deep into your experiences about how you were critical in execution for this project.  What tools did you use?  What was the problem you were trying to solve?  How much did you improve metrics?
* **Less Broad + More Defined Experiences**:  You have a lot of experiences, pick a couple from these buckets, maybe 3 projects where you can highlight.  Be very specific

Passing back to you for revision, feel free to request another review

DAQIANN committed:
Worked on improvements

I changed some more of the syntax and added more detail to my activities and experiences. I was wondering if I'm going in the right direction? I think I'm going to keep skills on the bottom.

DAQIANN committed:
Merge pull request #2 from DAQIANN/DAQIANN-resume-patch

Worked on improvements

alextakahashi commented:
Closing in favor of a new pull request
